www/index.html.spt

```diff
@@ -17,16 +22,39 @@ teams = website.db.all("""
   ORDER BY ctime DESC
 
 """)
+
 volume = website.db.one("SELECT volume FROM paydays ORDER BY ts_start DESC LIMIT 1", default=0)
 volume = int(round(volume, -2))
 
 tabs = OrderedDict()
+tabs['featured'] = {'teams': []}
 tabs['approved'] = {'teams': []}
 tabs['unreviewed'] = {'teams': []}
 tabs['rejected'] = {'teams': []}
 
-for team in teams:
-    tabs[team.status]['teams'].append(team)
+popular = []
+unpopular = []
+for project in teams:
+    tabs[project.status]['teams'].append(project)
+    if project.status == 'approved':
+      if project.nreceiving_from >= 5:
+        popular.append(project)
+      else:
+        unpopular.append(project)
+
+if len(teams) < 10:
+    tabs.pop('featured')
```

Comment:
This works. If you don't need to use the return value from `pop` then you can also do:

```python
del tabs['featured']
```

Just a little Python tip in case you didn't already know ... no need to change this here. :-)